example-gto/code/.github/workflows/gto-act-on-tags.yml

```diff
@@ -0,0 +1,46 @@
+name: Act on git tags produced by GTO
```

Comment:
what is the purpose of this action? let's say we would be actually using GTO, would we need all the things that this action does? 

can we skip full unshallow clone (it's not good if ppl will start copy-pasting this w/o need), running gto show, etc -- and get to `gto check-ref $GITHUB_REF` part that would do something with that REF? e.g runs a model or something?

Reply:
> Yes, I think we can add a usage example here

Checked, currently all models are just some non-existing references. To actually add a usage example, you would need to (1) create real models in buckets, (2) based on check-ref results actually apply those models.

I agree that this is important both for demo and for understanding whether we have a happy path here. I'm going to address this.